contracts/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.11;
+
+contract Migrations {
+  address public owner;
+  uint public last_completed_migration;
+
+  constructor() public {
```

Comment:
I think this function should be named "Migrations" if it is going to comply with v0.4.11. [Back then](https://solidity.readthedocs.io/en/v0.4.11/contracts.html#creating-contracts) constructors were the name of the contract.

Reply:
When I changed this line to read `function Migrations() public` it caused the following warning to be emitted when `yarn truffle test` was executed: `Warning: Defining constructors as functions with the same name as the contract is deprecated.`. Although this change may be academically correct it seems to go against the spirit of this pull request, which is to reduce overhead for newcomers to the repo, so I suggest leaving this particular line as it is.

Reply:
Ok, I believe that. It is a bit weird though because that would be invalid if you actually used a 04.11 compiler. Definitely better to avoid those warnings now though.